dolphinscheduler-alert-plugin/dolphinscheduler-alert-dingtalk/pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <artifactId>dolphinscheduler-alert-plugin</artifactId>
         <groupId>org.apache.dolphinscheduler</groupId>
-        <version>${revision}</version>
+        <version>1.3.6-SNAPSHOT</version>
```

Comment:
why did you replace the paramter revision to 1.3.6-SNAPSHOT？

Reply:
The original method will cause the release plug-in to not work normally, you can try it.
If you must do this, you must introduce the flatten plug-in, but I think it is completely unnecessary to introduce a new plug-in.